peachjam/models/save_document.py

```diff
@@ -105,6 +105,9 @@ class SavedDocument(models.Model):
     def __str__(self):
         return self.work.title
 
+    def get_absolute_url(self):
+        return self.work.frbr_uri
+
```

Comment:
@longhotsummer this fixes the citation link in the timeline

Reply:
This must use reverse (like `document.get_absolute_url`) otherwise the language prefix on multilingual sites won't be included